mobile/src/main/res/xml/preferences.xml

```diff
@@ -6,82 +6,103 @@
+                android:key="default_openhab_url"
+                android:summary="@string/settings_openhab_url_summary"
+                android:title="@string/settings_openhab_url"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_username"
+                android:summary="@string/settings_openhab_username_summary"
+                android:title="@string/settings_openhab_username"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textPassword"
+                android:key="default_openhab_password"
+                android:summary="@string/settings_openhab_password_summary"
+                android:title="@string/settings_openhab_password"/>
+        </PreferenceScreen>
+        <PreferenceScreen
+            android:title="@string/settings_openhab_alt_connection">
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_alturl"
+                android:summary="@string/settings_openhab_alturl_summary"
+                android:title="@string/settings_openhab_alturl"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textNoSuggestions"
+                android:key="default_openhab_remote_username"
+                android:summary="@string/settings_openhab_username_summary"
+                android:title="@string/settings_openhab_username"/>
+            <EditTextPreference
+                android:defaultValue="@string/empty_string"
+                android:dependency="default_openhab_demomode"
+                android:inputType="textPassword"
+                android:key="default_openhab_remote_password"
+                android:summary="@string/settings_openhab_password_summary"
+                android:title="@string/settings_openhab_password"/>
+        </PreferenceScreen>
+        <PreferenceScreen android:title="@string/settings_openhab_sslsettings">
+            <Preference
+                android:clickable="true"
+                android:defaultValue="@string/settings_openhab_none"
+                android:dependency="default_openhab_demomode"
+                android:key="default_openhab_sslclientcert"
+                android:summary=""
+                android:title="@string/settings_openhab_sslclientcert"/>
+            <Preference
+                android:clickable="true"
+                android:dependency="default_openhab_demomode"
```

Comment:
I'm not allowed to view the howto when demo mode is activated?

Reply:
I haven't changed that, it was the same dependency before, too, so yes :/

Reply:
I think the point is that SSL settings make no sense in demo mode. But my comment above also applies here: the dependencies need to be moved out to the PreferenceScreen, if those are going to be kept, as they won't work otherwise.